osu.Game.Tests/Visual/Online/TestSceneImageProxying.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Platform;
+using osu.Game.Graphics.Containers.Markdown;
+
+namespace osu.Game.Tests.Visual.Online
+{
+    public partial class TestSceneImageProxying : OsuTestScene
+    {
+        [Resolved]
+        private GameHost host { get; set; } = null!;
+
+        [Test]
+        public void TestExternalImageLink()
```

Comment:
How am I supposed to use these tests? Run locally and check console output? If so, mark them as not-CI-runnable.

Reply:
They are meant for local testing only, will mark them as ignored.

Reply:
That's fine, but I still don't know the expected outcome of each test. Local tests are only useful if it's easy for the person running them to know what they are looking for.

Reply:
I've rewrote the tests to assert the end behaviour. I've intentionally left it as CI runnable since there's no reason not to, unless it's bad to have a test which depends on external lookups.